src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -258,6 +258,10 @@ const theme = {
   },
 
   grid: 8,
+
+  // Relative font sizes
+  fontSizeSmaller: '0.9em',
```

Comment:
Nit: Might be better to name this as `fontSizeRelativeSmaller` so its purpose would be clear when it is being used elsewhere.